### ISSUE TITLE
TypeError: Cannot read property 'pathname' of undefined

### ISSUE THREAD

**Issue by philrz** [brimdata/brim#1648]
To repro, I started by running GA Brim tagged `v0.24.0` and made a Space by importing the attached
[ifconfig.pcapng.gz](https://github.com/brimdata/brim/files/6490268/ifconfig.pcapng.gz) (after uncompressing).

### This next step is very important, as the bug does not repro without it:

I then clicked to a new tab and closed the original tab that showed the contents of the Space after import was complete. That is, this bug does _not_ trigger if I leave the app on a tab that's viewing the Space before I quit.

That's all shown in this first video.

https://user-images.githubusercontent.com/5934157/118416658-a7513000-b665-11eb-971f-fcc2b4ce9dfa.mp4

While in this state with no Space selected on the current tab, I quit the app and upgrade to a new release, specifically Brim commit 43b0ed5 via nightly macOS build [https://storage.googleapis.com/brimsec-releases/brim/v0.25.0-prerelease-43b0ed51.0/macos/Brim-0.25.0-prerelease-43b0ed51.0.dmg](https://storage.googleapis.com/brimsec-releases/brim/v0.25.0-prerelease-43b0ed51.0/macos/Brim-0.25.0-prerelease-43b0ed51.0.dmg). As you can see in the next video, as soon as the app is done launching, I'm brought to a crash screen.

https://user-images.githubusercontent.com/5934157/118416668-b506b580-b665-11eb-8be4-b8850b5fd6af.mp4

The text from the crash:

```
TypeError: Cannot read property 'pathname' of undefined
    at TabsRouter.render (/Applications/Brim.app/Contents/Resources/app.asar/dist/app/router/tabs-router.js:137:55)
    at finishClassComponent (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:17160:31)
    at updateClassComponent (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:17110:24)
    at beginWork (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:18620:16)
    at HTMLUnknownElement.callCallback (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:188:14)
    at Object.invokeGuardedCallbackDev (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:237:16)
    at invokeGuardedCallback (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:292:31)
    at beginWork$1 (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:23203:7)
    at performUnitOfWork (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:22157:12)
    at workLoopSync (/Applications/Brim.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:22130:22)
```

**Comment by philrz** [comment]
In fact, I've since found I can repro this even easier by just starting out in `v0.24.0` with no Space data at all, then just upgrade straight to the newer version and launch.

**Comment by philrz** [comment]
Verified in Brim commit bd5d993 by using scratch macOS build [https://storage.googleapis.com/brimsec-releases/brim/v0.25.0-prerelease-bd5d9930.0/macos/Brim-0.25.0-prerelease-bd5d9930.0.dmg.](https://storage.googleapis.com/brimsec-releases/brim/v0.25.0-prerelease-bd5d9930.0/macos/Brim-0.25.0-prerelease-bd5d9930.0.dmg.)

@jameskerr informed me that this issue was ultimately not specific to Space migration, but since that's how I originally stumbled onto it, I've verified in the following video using the same repro steps. It picks up where I launch into the updated Brim after having closed the `v0.24.0` one while on a single blank tab.

https://user-images.githubusercontent.com/5934157/118563427-615aa180-b723-11eb-9483-6e67b296d8b7.mp4

Thanks @jameskerr!
